docs/src/modules/components/MarkdownElement.js

```diff
@@ -56,32 +55,42 @@ const Root = styled('div')(({ theme }) => ({
   '& h1': {
     ...theme.typography.h3,
     fontSize: 40,
+    fontFamily: ['"PlusJakartaSans"'].join(','),
```

Comment:
```suggestion
    fontFamily: ['"PlusJakartaSans-ExtraBold"'].join(','),
```

it should have `ExtraBold` as a suffix